doc/release/upcoming_changes/25437.new_feature.rst

```diff
@@ -0,0 +1,8 @@
+`numpy.linalg.martrix_rank`, `numpy.sort` and `numpy.argsort` new parameters
+----------------------------------------------------------------------------
+
+New keyword parameters were added to improve array API compatibility:
+
+* ``rtol`` keyword parameter was added to `numpy.linalg.martrix_rank`.
```

Comment:
The fact that this changes the default, does need it is own release note, as it is a breaking change.

Reply:
Note that this was not addressed, still need a release note nothing the BC break.

Reply:
I don't think it changes the default.

Before default `tol` was defined as:
```python
if tol is None:
    tol = (
        S.max(axis=-1, keepdims=True) *
        max(A.shape[-2:]) *
        finfo(S.dtype).eps
    )
```

And right now it's:
```python
if rtol is None:
    rtol = max(A.shape[-2:]) * finfo(S.dtype).eps
if tol is None:
    tol = S.max(axis=-1, keepdims=True) * rtol
```
As `rtol=None` and `tol=None` by default, then the default didn't change. Or am I missing something?

Reply:
Right, sorry... it does change for `pinv`, but not matrix rank.